src/qt/editor/editorgrapicsitem.cpp

```diff
@@ -91,8 +91,27 @@ void EditorGraphicsItem::showPortInfo(QGraphicsSceneHelpEvent* e, Port* port) co
 
     using P = Document::PathComponent;
 
-    auto doc = port->getInfo();
-    auto b = doc.get({P("html"), P("body")});
+    Document desc{};
+    auto html = desc.append("html");
+    html.append("head").append("style", R"(
```

Comment:
global qss style?

Reply:
This post from 2006 indicates that it is not possible. https://www.qtcentre.org/threads/55843-Style-sheet-for-QGraphicsItem

Reply:
this is not for the graphics item but the document that is displays